packages/wallet-backend-update/src/account/validation.ts

```diff
@@ -7,7 +7,7 @@ export const accountSchema = z.object({
   })
 })
 
-export const fundSchema = z.object({
+export const fundsObject = {
```

Comment:
I think we can keep it as `fundSchema` and use them for both actions or just re-export `withdrawFundsSchema` without creating an object and then passing it to `zod`. E.g.:

```ts
export const fundSchema = z.object({ /* ... */ })
export const withdrawFundsSchema = fundSchema
```